README.md

```diff
@@ -1,37 +1,78 @@
 # cppdep
 [![Build Status](https://travis-ci.org/cgilling/cppdep.svg?branch=master)](https://travis-ci.org/cgilling/cppdep)
 
-Playing Around With C++ Dependencies and simple naive compilation
-
-## Warning
-
-This is still in active development. Where it is now in a working state, the config file and command line interface is likely to change in the near future.
+Simple compilation of an arbitrary C++ source tree with autmatic dependency discovery.
```

Comment:
Think you could talk up the speed a bit more too, i.e. `Simple, fast compilation...` etc.